core/sets.gms

```diff
@@ -754,123 +754,131 @@ sets
    ext_regi "extended regions list (includes subsets of H12 regions)"
       /
```

Comment:
I don't understand these changes to core/sets.gms. Do they come from using REMIND-EU? I think they should not be pushed to the repository.

Reply:
Keine Ahnung, woher das kam. ich habe das rausgenommen.